.github/workflows/operationcheck.yml

```diff
@@ -0,0 +1,15 @@
+# Checks client builder for operations for each type of object.
+name: operation_check
```

Comment:
I have some questions about a couple things:

1. Having > 1 small checks vs. 1 big check. I can see the appeal of small checks in say a PR view. But the problem there is the inability to run 1 distinct command, locally, that ensures all these boxes are hit. Should not need GHA to do this.
2. We already have a Savant build process for this artifact. I think we should include this type of logic in 1 of those targets such that we have 1 coherent way to build and validate. I don't think the shell script needs to be re-written in Groovy, but I think we should be calling it from there. The pattern we seem to be settling into is running this from the `int` target. Thoughts?

Reply:
oops, do I need to revert this @wied03 ?